variables.tf

```diff
@@ -66,55 +66,55 @@ variable "visibility" {
 variable "has_issues" {
   description = "(Optional) Set to true to enable the GitHub Issues features on the repository. (Default: false)"
   type        = bool
-  default     = null
+  default     = false
 }
 
 variable "has_projects" {
   description = "(Optional) Set to true to enable the GitHub Projects features on the repository. Per the github documentation when in an organization that has disabled repository projects it will default to false and will otherwise default to true. If you specify true when it has been disabled it will return an error.  (Default: false)"
   type        = bool
-  default     = null
+  default     = false
 }
 
 variable "has_wiki" {
   description = "(Optional) Set to true to enable the GitHub Wiki features on the repository. (Default: false)"
   type        = bool
-  default     = null
+  default     = false
 }
 
 variable "allow_merge_commit" {
   description = "(Optional) Set to false to disable merge commits on the repository. (Default: true)"
   type        = bool
-  default     = null
+  default     = true
 }
 
 variable "allow_squash_merge" {
   description = "(Optional) Set to true to enable squash merges on the repository. (Default: false)"
   type        = bool
-  default     = null
+  default     = false
 }
 
 variable "allow_rebase_merge" {
   description = "(Optional) Set to true to enable rebase merges on the repository. (Default: false)"
   type        = bool
-  default     = null
+  default     = false
 }
 
 variable "delete_branch_on_merge" {
   description = "(Optional) Whether or not to delete the merged branch after merging a pull request. (Default: false)"
   type        = bool
-  default     = null
+  default     = false
 }
 
 variable "has_downloads" {
   description = "(Optional) Set to true to enable the (deprecated) downloads features on the repository. (Default: false)"
   type        = bool
-  default     = null
+  default     = false
 }
 
 variable "auto_init" {
```

Comment:
Those kind of code changes require adjustments in the `main.tf` also since we're checking for null there. Also, it should be added to the `Changelog.md` - may I ask you why we decided to implement those changes?

Reply:
The documentation consistently put `false` where `variables.tf` had `null`. It looked like a mistake to me, like someone forgot to add the defaults in `variables.tf`. But it seems it was the other way around and the documentation was wrong. I will change it back and change the documentation.

Reply:
yeah that seems to be the case! Sorry for the confusion - this was obviously already faulty in the first place! Thank you!

Reply:
Ah, I had a look at main.tf and it sets the defaults there. Now I get why it's like that.

Reply:
oh that makes sense :) so eventually I was wrong also =D Good catch @kassianh 

The question is how shall we handle this in the docs, @mariux ?

Reply:
Actually the documentation is correct since if no default is set it will default to whatever the documentation says. It just does so in `main.tf` which I did not realize.

Reply:
makes sense! Thanks for the clarification!

Reply:
Eg:
In `main.tf`:
`has_wiki = var.has_wiki == null ? lookup(var.defaults, "has_wiki", false) : var.has_wiki`

And in the documentation:
`variable "has_wiki" {
        type        = bool
        default     = false
        description = <<-END
          Set to true to enable the GitHub Wiki features on the repository.
        END
}`

So it does set it to `false` if it can't find `has_wiki` in `var.defaults`, hence the documentation is correct.